testbed/tests/test_window.py

```diff
@@ -487,6 +487,21 @@ async def test_as_image(main_window, main_window_probe):
     main_window_probe.assert_image_size(screenshot.size, main_window_probe.content_size)
 
 
+# Test the `origin`, `position` and `screen_position`.
+async def test_screen(main_window, main_window_probe):
+    """The window can be relocated to another screen, using both absolute and relative screen positions."""
+
+    if main_window.screen.origin == (0, 0):
+        if toga.platform.current_platform in {"android", "iOS", "textual"}:
+            pytest.xfail("Window.position is non functional on current platform.")
+        initial_position = main_window.position
+        main_window.position = (200, 200)
+        assert main_window.position != initial_position
+        # position and screen_position should be equal on screen with origin (0, 0)
+        assert main_window.position == (200, 200)
+        assert main_window.screen_position == (200, 200)
```

Comment:
Tests should always restore original conditions - this test needs to either operate on a secondary window, or clean up after itself. I suspect a secondary window will be easier/cleaner - and would also allow you to move the window across "every" screen (even though there may only be 1 screen in some test conditions)

Reply:
Done.